packages/syft/src/syft/types/errors.py

```diff
@@ -0,0 +1,55 @@
+# stdlib
+from typing import Any
+
+# relative
+from ..service.context import AuthedServiceContext
+from ..service.user.user_roles import ServiceRole
+
+
```

Comment:
How do we send this between worker and backend or other nodes?
Should we consider having a single message exception and then keeping the public / private as kind of twin somewhere?